<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loyalcred</class>
 <widget class="QMainWindow" name="loyalcred">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>55</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DM Sans</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOYALTY CREDS</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>57</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/creds.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>600</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 11pt &quot;DM Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Redeem Creds</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>341</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/cred1.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 14pt &quot;DM Sans&quot;;</string>
    </property>
    <property name="text">
     <string>CRED BALANCE:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 14pt &quot;DM Sans&quot;;</string>
    </property>
    <property name="text">
     <string>CREDS REDEEMED: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 14pt &quot;DM Sans&quot;;</string>
    </property>
    <property name="text">
     <string>CRED LEVEL:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 11pt &quot;DM Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Creds are our way of giving our nomadics the best in all deals! Every time you make your travels easier</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>661</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 11pt &quot;DM Sans&quot;;</string>
    </property>
    <property name="text">
     <string> through nomadic account, you earn creds which can then be redeemed to avail many offers!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accGoback">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>690</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;DM Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Go back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="source02.qrc"/>
 </resources>
 <connections/>
</ui>
